content/zh/blog/_posts/2020-03-25-kubernetes-1.18-release-announcement.md

```diff
@@ -298,7 +298,7 @@ The simplest way to get involved with Kubernetes is by joining one of the many [
 -->
 - 在 Twitter 上关注我们 [@Kubernetesio](https://twitter.com/kubernetesio)，了解最新动态
 - 在 [Discuss](https://discuss.kubernetes.io/) 上参与社区讨论 
-- 加入 [Slack](http://slack.k8s.io/) 上的社区
+- 加入 [Slack](https://slack.k8s.io/) 上的社区
 -  在[Stack Overflow](http://stackoverflow.com/questions/tagged/kubernetes)提问（或回答）
```

Comment:
```suggestion
-  在[Stack Overflow](https://stackoverflow.com/questions/tagged/kubernetes)提问（或回答）
```

FYI: This suggested change of mine is already suggested in the PR #28703.

How about merge the PRs #28702 and #28703 ?